src/index.ts

```diff
@@ -10,15 +11,18 @@ import {
 const GITHUB_TOKEN = getInput('GITHUB_TOKEN');
 
 const octokit = new GitHub(GITHUB_TOKEN);
+const GITHUB_LOGIN = getInput('GITHUB_LOGIN');
 
 const main = async (): Promise<void> => {
+  info(`The bot name is ${GITHUB_LOGIN}`);
```

Comment:
```suggestion
  info(`Automatic merges enabled for a bot with GitHub login ${GITHUB_LOGIN}.`);
```

Reply:
strictly speaking, this does not really tie to any bot, just a Github login.

Reply:
https://github.com/ridedott/dependabot-auto-merge-action/pull/233/commits/6ccdba337490234b2a3d3a9d346c82c38a88185c WDYT?